cmd/geth/dbcmd.go

```diff
@@ -158,6 +161,22 @@ WARNING: This is a low-level operation which may cause database corruption!`,
 		Description: `This command sets a given database key to the given value. 
 WARNING: This is a low-level operation which may cause database corruption!`,
 	}
+	dbGetSlotsCmd = cli.Command{
+		Action:    utils.MigrateFlags(dbDumpTrie),
+		Name:      "dumptrie",
+		Usage:     "Show the storage key/values of a given storage trie",
+		ArgsUsage: "<hex-encoded storage trie root>",
```

Comment:
We should add the description to the `ArgsUsage` for the optional `seek` and `maxCount`.

Reply:
done!